examples/ios/cubic-demo/ViewController.swift

```diff
@@ -140,36 +140,38 @@ class ViewController: UIViewController, UIGestureRecognizerDelegate, CubicManage
                 self.createCubicManager(host: ViewController.CUBIC_HOST,
                                         port: ViewController.CUBIC_PORT,
                                         useTLS: useTLS)
+                
             }
            
             
         }
     }
     
     @IBAction func recordClickDown(sender:UIButton)  {
-        print("recordClick Down")
-        if cubicManager.isAuthorized() {
+        if cubicRecorder.isAuthorized() {
             self.recordButton.tintColor = UIColor.red
-            self.cubicManager.record()
+            self.cubicRecorder.startStream()
         } else {
-            self.cubicManager.requestAccess { (granted) in
+            self.cubicRecorder.requestAccess { (granted) in
                  print("recordClick \(granted)")
             }
         }
     }
     
     @IBAction func recordClickUp(sender:UIButton)  {
-        print("recordClick Up")
-        self.cubicManager.stop()
+        self.cubicRecorder.stopStream()
         self.recordButton.tintColor = self.view.tintColor
     }
+    
     func streamCompletion(_ result: Cobaltspeech_Cubic_RecognitionResponse?) {
-        
+         printResult(response: result,replace:false)
+    }
+    func streamingHandler(result: Cobaltspeech_Cubic_RecognitionResponse) {
+              self.printResult(response: result,replace:false)
     }
-    
     // MARK: - Response processing
     
-    private func printResult(response: Cobaltspeech_Cubic_RecognitionResponse?) {
+    private func printResult(response: Cobaltspeech_Cubic_RecognitionResponse?,replace:Bool) {
```

Comment:
Is there a general code linter that should be used for swift? This function appears to have more whitespace, not sure if that's something we could catch.